javascript/javascript1/week2/addCandy.js

```diff
@@ -0,0 +1,44 @@
+let boughtCandyPrices = [];
+let amountToSpend = Math.random() * 100;
+function addCandy(candyType, weight) {
+  if (candyType === "sweet") {
+    let price1 = 0.5 * weight;
+    boughtCandyPrices.push(price1);
+    return;
+  }
+  if (candyType === "Chocolate") {
+    let price2 = 0.7 * weight;
+    let n = boughtCandyPrices.push(price2);
+    return;
+  }
+  if (candyType === "Toffee") {
+    let price2 = 1.1 * weight;
+    let n = boughtCandyPrices.push(price2);
+    return;
+  }
+  if (candyType === "Chewing-gum") {
+    let price2 = 0.03 * weight;
+    let n = boughtCandyPrices.push(price2);
+    return;
+  }
+}
+addCandy("sweet", 20);
+addCandy("Chocolate", 100);
+console.log(boughtCandyPrices);
+console.log(amountToSpend);
+
+function canBuyMoreCandy() {
+  let total = 0;
+  for (let i = 0; i < boughtCandyPrices.length; i++) {
+    total = total + boughtCandyPrices[i];
+  }
```

Comment:
- I like this. other students used `while` and I think your way is easier.
- You will also learn later another easier way using `reduce` on arrays